src/demo.css

```diff
@@ -0,0 +1,11 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand&display=swap');
+
+*{ 
+  font-family: 'Quicksand', sans-serif;
+} 
+
+body {
```

Comment:
Try and avoid using the CSS files for anything other then fonts.
Put this style in the tsx file.

Reply:
Updated per comment. CSS file now only contains font information. Styling for `body` has been moved to `MainScene.tsx`.